src/Hedgehog.Experimental/Gen.fs

```diff
@@ -277,18 +277,19 @@ module GenX =
   let increasing4 (g : Gen<'a * 'a * 'a * 'a>) : Gen<'a * 'a * 'a * 'a> =
     g |> sorted4 |> distinct4
 
-  /// Generates a tuple of datetimes where the range determines the minimum
+  /// Generates a tuple of datetimes where the dayRange determines the minimum
   /// and maximum number of days apart. Positive numbers means the datetimes
   /// will be in increasing order, and vice versa.
-  let dateInterval (dayRange : Range<int>) : Gen<DateTime * DateTime> =
+  let dateInterval (range : Range<DateTime>) (dayRange : Range<int>) : Gen<DateTime * DateTime> =
```

Comment:
Hm, this is one way to go about it, but another way is to simply use a default range, e.g the exponential range mentioned in the previous comment. What do you think users of this generator need? Specifying the date range hasn't been possible until now anyway, but if you think it's useful, I'm open to this method. It will require changes in code that uses it, though.

Reply:
Agreed - I don't think breaking this API is warranted.